cryptoki/tests/basic.rs

```diff
@@ -364,6 +366,100 @@ fn session_find_objects() {
     assert_eq!(found_keys.len(), 9);
 }
 
+#[test]
+#[serial]
+fn session_objecthandle_iterator() {
```

Comment:
```suggestion
fn session_objecthandle_iterator() -> testresult::TestResult {
```

(and `Ok(())` at the end)

And then (almost) all `unwrap`s can just be converted to `?`.

Reply:
That's a good one!